examples/build_overrides/mbedtls.gni

```diff
@@ -15,4 +15,5 @@
 declare_args() {
   # Root directory for mbedTLS.
   mbedtls_root = "//third_party/connectedhomeip/third_party/mbedtls"
+  mbedtls_repo = "//third_party/connectedhomeip/third_party/mbedtls"
```

Comment:
Why exactly do we need both `mbedtls_root` and `mbedtls_repo`?

Reply:
It was the only solution I could find to differentiate between the mbedtls gn files
`third_party/mbedtls/BUILD.gn` & `third_party/mbedtls/mbedtls.gni` and the source files.
This way you can use the gn build files through `mbedtls_root` and the source files through `mbedtls_repo`.
`mbedtls_repo` can be overwritten from `--args` to point to other mbedtls source files.

I'm not sure if it is the best solution though. Do you have a better one to suggest? Thank you.